### DISCUSSION TITLE
Getting error when one of the service is starting up

### DISCUSSION THREAD

**Discussion by ndave25** [frankframework/frankframework#6319] [Q&A]
Hello all,

I need urgent help in resolving the below error:

ERROR [nl.nn.adapterframework.receivers.GenericReceiver] (SimpleAsyncTaskExecutor-15) caught Exception retrieving message, will continue retrieving messages in [4] seconds: java.util.NoSuchElementException
        at deployment.xportal.ear.xPortalWeb.war//org.apache.commons.lang.text.StrTokenizer.next(StrTokenizer.java:539)
        at deployment.xportal.ear.xPortalWeb.war//nl.nn.adapterframework.jdbc.MessageStoreListener.getRawMessage(MessageStoreListener.java:109)
        at deployment.xportal.ear.xPortalWeb.war//nl.nn.adapterframework.receivers.PullingListenerContainer$ListenTask.run(PullingListenerContainer.java:179)
        at java.base/java.lang.Thread.run(Thread.java:829)

We have 3 adapters, 2 of them are starting up without any issue but of them is having the above error. Can you please me why the above error is coming?

Sorry if this is a silly question but I am just supporting this application and the developer has just left the company.

**Comment by nielsm5** [comment]
Which version are you using?

**Reply by ndave25** [reply]
ibis-adapterframework-core-7.2

**Reply by nielsm5** [reply]
Oeh, that's quite old, although not impossible, it makes debugging this quite a bit more difficult. It looks like it's trying to read a value from the database (parsed as CSV so comma separated values), and by the looks of it the value (from the database) is empty.

**Reply by ndave25** [reply]
This are the DB table columns:

MESSAGEKEY                                NOT NULL NUMBER(10)
TYPE                                              CHAR(1 CHAR)
SLOTID                                          VARCHAR2(100 CHAR)
HOST                                             VARCHAR2(100 CHAR)
MESSAGEID                                   VARCHAR2(100 CHAR)
CORRELATIONID                           VARCHAR2(256 CHAR)
MESSAGEDATE                              TIMESTAMP(6)
COMMENTS                                  VARCHAR2(1000 CHAR)
MESSAGE                                      BLOB
EXPIRYDATE                                  TIMESTAMP(6)
LABEL                                            ARCHAR2(100 CHAR)

I think it the Message coming as null

**Reply by jacodg** [reply]
@ndave25 would it be possible to setup a Teams meeting so we can discuss it? I think we need a bit more context. See my contact information at https://wearefrank.nl/contact/. Feel free to mail me or use my mobile number.
